src/fleet/azext_fleet/commands.py

```diff
@@ -40,15 +40,15 @@ def load_command_table(self, _):
         g.custom_command("update", "update_fleet", supports_no_wait=True)
         g.custom_show_command("show", "show_fleet")
         g.custom_command("list", "list_fleet")
-        g.custom_command("delete", "delete_fleet", supports_no_wait=True)
+        g.custom_command("delete", "delete_fleet", supports_no_wait=True, confirmation=True)
```

Comment:
It's a breaking change for the current users. Please add history notes for it.

Reply:
I've updated the version and notes .